.github/workflows/actions.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           stable: 'true'
-          go-version: '1.16' # The Go version to download (if necessary) and use.
+          go-version: '1.17' # The Go version to download (if necessary) and use.
```

Comment:
Hi @baconwill , we'd better move this to a separate PR, maybe call it something like **update GitHub workflow go version**?
It is better to keep one PR for one purpose and keep the PR changes smaller.